### DISCUSSION TITLE
这盛世如你所愿

### DISCUSSION THREAD

**Discussion by wedxz** [WeNeedHome/SummaryOfLoanSuspension#367] [General]
此生无悔入华夏,来世还做中国人

**Comment by wedxz** [comment]
中华民族伟大复兴==全民负债==收税收到三十年以后

**Reply by wedxz** [reply]
跟房贷比起来社保才有多少

**Reply by lance111l** [reply]
美国的贷款和美债也一样  美国习惯用未来的钱

**Comment by Coder-LinShun** [comment]
怕是要揭棺而起

**Reply by wedxz** [reply]
没有到生死存亡的时候,是不可能发生这种情况,老百姓们还能忍

**Reply by Wesleylr** [reply]
没那么容易，中国人还是很温顺的，参考历史，大规模反抗其实都是在吃不饱的时候才会反抗，现在生产力如此发达除非步步走错不然不太可能的

**Reply by 0827cg** [reply]
> 没那么容易，中国人还是很温顺的，参考历史，大规模反抗其实都是在吃不饱的时候才会反抗，现在生产力如此发达除非步步走错不然不太可能的

聪明人喜欢做聪明事, 背后搓使底层农民起来反抗。其实中国底层人民并不是很温顺

**Reply by miaodi** [reply]
都想着不劳而获，所以还是跑吧。粪坑没救

**Comment by yalo-ddaa** [comment]
少发点灌水贴，留给有用的信息。

**Reply by wedxz** [reply]
有的人总是意识不到问题出在哪里

**Reply by liuwenwu520** [reply]
有的人永远只能打嘴炮

**Comment by qq912276337** [comment]
再苦一苦大家

**Comment by wedxz** [comment]
21年郑州大水，外国记者采访报道，郑州人民围追堵截全城搜捕，禁止外部势力采访受灾人，22年河南储户存款在乡镇银行取不出，自己闹了一个月有余，国内没有一家媒体积极报道，无奈打出英文横幅和毛主席头像，可惜外媒记者都被你们举报光了，再也没人可以替你们发声，你们终究还是没明白自己输在哪里

**Reply by gabriel-shu** [reply]
郑州人民的名义？

**Reply by shadowlgd** [reply]
如果这次能让人民觉醒不再上当也行。

**Reply by yaojun170** [reply]
人民的选择

**Reply by volans19** [reply]
善恶到头终有报

**Reply by lettermessenger** [reply]
你挑的嘛！偶像！

**Reply by blisee** [reply]
输在哪里啊？

**Reply by blisee** [reply]
你意思外国媒体报道了问题就解决了？

**Reply by wwpp9900** [reply]
内部矛盾和外部矛盾两回事，你和你爸吵架，我去打你爸，你同意不同意？

**Reply by wedxz** [reply]
> 内部矛盾和外部矛盾两回事，你和你爸吵架，我去打你爸，你同意不同意？

你爸会让你掏空六个钱包凑的首付款买的房子烂尾？

**Reply by wwpp9900** [reply]
别转移话题，你同意不同意？

还有：
我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！

**Reply by wedxz** [reply]
> 别转移话题，你同意不同意？
> 
> 还有：
> 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！

烂尾楼是西方强盗搞出来的？

**Reply by wwpp9900** [reply]
> > 别转移话题，你同意不同意？
> > 
> > 还有：
> > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> 
> 烂尾楼是西方强盗搞出来的？

你可真会转移话题，我问你同意不同意，你不回复！
我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。

说的话完全没有逻辑可言！

你是个什么成分已经很清楚了！！！

那么请你回复我第一个问题，你同不同意？？？

**Reply by wedxz** [reply]
> > > 别转移话题，你同意不同意？
> > > 
> > > 还有：
> > > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> > 
> > 烂尾楼是西方强盗搞出来的？
> 
> 你可真会转移话题，我问你同意不同意，你不回复！
> 我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。
> 
> 说的话完全没有逻辑可言！
> 
> 你是个什么成分已经很清楚了！！！
> 
> 那么请你回复我第一个问题，你同不同意？？？
> 

可是他不是我爸啊，你这个比喻不恰当，我回答的有什么意义

**Reply by wedxz** [reply]
> > > > 别转移话题，你同意不同意？
> > > > 
> > > > 还有：
> > > > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> > > 
> > > 烂尾楼是西方强盗搞出来的？
> > 
> > 你可真会转移话题，我问你同意不同意，你不回复！
> > 我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。
> > 
> > 说的话完全没有逻辑可言！
> > 
> > 你是个什么成分已经很清楚了！！！
> > 
> > 那么请你回复我第一个问题，你同不同意？？？
> > 
> 
> 可是他不是我爸啊，你这个比喻不恰当，我回答的有什么意义

哦，不对，应该是我没这样欺压百姓的父亲

**Reply by wwpp9900** [reply]
> > > > 别转移话题，你同意不同意？
> > > > 
> > > > 还有：
> > > > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> > > 
> > > 烂尾楼是西方强盗搞出来的？
> > 
> > 你可真会转移话题，我问你同意不同意，你不回复！
> > 我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。
> > 
> > 说的话完全没有逻辑可言！
> > 
> > 你是个什么成分已经很清楚了！！！
> > 
> > 那么请你回复我第一个问题，你同不同意？？？
> > 
> 
> 可是他不是我爸啊，你这个比喻不恰当，我回答的有什么意义


别回了，你以为你能带多少节奏？

就这智商这逻辑思维还上gethub教一群玩代码的怎么做事，笑话！
你表演的场地在推特，在youtube！！！

**Reply by wedxz** [reply]
> > > > > 别转移话题，你同意不同意？
> > > > > 
> > > > > 还有：
> > > > > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> > > > 
> > > > 烂尾楼是西方强盗搞出来的？
> > > 
> > > 你可真会转移话题，我问你同意不同意，你不回复！
> > > 我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。
> > > 
> > > 说的话完全没有逻辑可言！
> > > 
> > > 你是个什么成分已经很清楚了！！！
> > > 
> > > 那么请你回复我第一个问题，你同不同意？？？
> > > 
> > 
> > 可是他不是我爸啊，你这个比喻不恰当，我回答的有什么意义
> 
> 
> 别回了，你以为你能带多少节奏？
> 
> 就这智商这逻辑思维还上gethub教一群玩代码的怎么做事，笑话！
> 你表演的场地在推特，在youtube！！！

你以为你爸就仅仅那些贪官吗？你不知道有个学校的犯罪率第一吗？看看那些官员有多少是中央党校出来的？会动脑子的话想想吧，不会的话就算了

**Reply by wwpp9900** [reply]
> > > > > > 别转移话题，你同意不同意？
> > > > > > 
> > > > > > 还有：
> > > > > > 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！
> > > > > 
> > > > > 烂尾楼是西方强盗搞出来的？
> > > > 
> > > > 你可真会转移话题，我问你同意不同意，你不回复！
> > > > 我说干死导致烂尾楼的那些人，也要干死西方强盗，你和我扯烂尾楼是谁搞出来的。
> > > > 
> > > > 说的话完全没有逻辑可言！
> > > > 
> > > > 你是个什么成分已经很清楚了！！！
> > > > 
> > > > 那么请你回复我第一个问题，你同不同意？？？
> > > > 
> > > 
> > > 可是他不是我爸啊，你这个比喻不恰当，我回答的有什么意义
> > 
> > 
> > 别回了，你以为你能带多少节奏？
> > 
> > 就这智商这逻辑思维还上gethub教一群玩代码的怎么做事，笑话！
> > 你表演的场地在推特，在youtube！！！
> 
> 你以为你爸就仅仅那些贪官吗？你不知道有个学校的犯罪率第一吗？看看那些官员有多少是中央党校出来的？会动脑子的话想想吧，不会的话就算了

都叫你别回了，你推特上面复制的那些三瓜两枣在这里带不来颜色革命！
是吧，呆蛙，呱呱叫有什么用！蛙就是蛙，老老实实回到你的推特上面去还能多跳跳！
迟早被下酒，何必呢！
我估计你也看不懂我上面说的话，算了，肚子饿了的时候再去买酒，应该挺香！

**Reply by 0827cg** [reply]
`郑州人民围追堵截全城搜捕`: 是全城么?, 老少皆上.? 还是只有那么一群人.?你能确定是什么人么?，他们能代表全城.?
`外媒记者都被你们举报光了，再也没人可以替你们发声`: 记者记者，记者是中立的. [怕你找不到，给你贴个链接吧，小娃子](https://cn.nytimes.com/china/20220712/china-bank-protest/?_ga=2.130332365.351744065.1658194962-885433704.1658194962)

**Reply by TheGreateJohny** [reply]
> 内部矛盾和外部矛盾两回事，你和你爸吵架，我去打你爸，你同意不同意？

人家说的好像是以前你爸打你，隔壁老王说别打了别打了，然后你和老王说滚一边去，我们这是“内部矛盾”，管你屁事。

现在你爸打你，没有外人参合了，只有你爸打你了。

**Reply by renjianbingqi** [reply]
今天720 一周年，风控半个月前就开始了。

**Reply by maxwillma** [reply]
> 别转移话题，你同意不同意？
> 
> 还有： 我希望所有的烂尾的老百姓都能站出来，干死那些贪婪的狗东西。但是我也希望中国干死西方这些强盗！我站的是正义，不是像你这样只知道带节奏，唯恐天下不乱，为了实现自己肮脏目的无所不用其极！！！

你就知道  干    你这个中二

**Comment by Chain-Yu** [comment]
此生无悔入华夏,来世还得再想想

**Reply by wedxz** [reply]
我们是最后一带,谢谢

**Comment by ChinaChenMingQuan** [comment]
确实是如你所愿呀。除非你不是上层的百姓，而是我们这些底层“畜生”。什么？你说中层算什么？它们顶多算是管理我们这些“畜生”的奴隶罢了。

**Reply by wedxz** [reply]
好好审题
现在价值观基本等于金钱观,没有任何的道德,法律,尊严可谈,为了钱不择手段,与其说成为金钱的奴隶,不如说成为权力阶层所掌控下的奴隶

**Reply by ppinkiris** [reply]
按你的逻辑不会觉得自己在灯塔国拿张选票就能当人了吧？所以美国为什么不禁枪？

**Comment by songning1992** [comment]
是不是少了个括弧补充说明？(仅限港澳台)

**Comment by dongyc519** [comment]
一定要区分d和国。我们国家的人民是伟大的人民，我们和世界其他地方的人一样，有平等的权利。凡是受到区域的这个d的限制，那d就是非法的。

**Reply by pfdh** [reply]
共产党的理念是没有错的，但奈何一些人不干人事！

**Reply by Pinruyi** [reply]
> 共产党的理念是没有错的，但奈何一些人不干人事！

皇帝是好的就奸臣太多？

**Reply by wedxz** [reply]
> 共产党的理念是没有错的，但奈何一些人不干人事！

所以就可以坏的理所当然？

**Reply by clleo888** [reply]
共产党的理念就是骗, 你认为骗没错?

**Reply by Cegazhos** [reply]
> 共产党的理念是没有错的，但奈何一些人不干人事！

看看你党自己的党史吧 还甩锅呢 连承认都不敢。

**Comment by luowenfeng** [comment]
此生无, 悔入华夏

**Reply by shawn-yee** [reply]
一个字,绝.gif

**Reply by lance111l** [reply]
弱智的表现就是 抱怨自己的出生 自己的父母 甚至自己的朋友    
这是没希望的

**Comment by zhangf115** [comment]
那你快走啊，还赖在这干啥？

**Reply by ruo-xi** [reply]
经典二极管，知道为什么有人踩你吗？

**Reply by MYWmiss** [reply]
你TM不识字吗？人家在夸赞祖国，你让人家走？你的脑子呢？

**Reply by jiushi555** [reply]
我在想，要是在文革，你会不会把你妈也举报了？

**Comment by Novite5** [comment]
问题是全球都一样，只是有的国家的人民想当房奴都没机会。

**Reply by wedxz** [reply]
> 问题是全球都一样，只是有的国家的人民想当房奴都没机会。

比烂吗

**Reply by zhw173** [reply]
房奴和房奴是不一样的，人家交房才开始还房贷，人家烂尾可以终止合同，比啥比

**Reply by Pinruyi** [reply]
咱能上外网动动手指搜一下也行，没必要在这玩阿Q

**Reply by selphy65** [reply]
@Pinruyi 哈哈哈

**Reply by clleo888** [reply]
自己有问题就说全世界都有问题, 行呀!
认认真真去改善自己的问题有这麽难吗?

**Reply by wedxz** [reply]
> 自己有问题就说全世界都有问题, 行呀!
> 认认真真去改善自己的问题有这麽难吗?

他们的逻辑就是。都有问题等于没问题

**Reply by lance111l** [reply]
> 房奴和房奴是不一样的，人家交房才开始还房贷，人家烂尾可以终止合同，比啥比

你自己签合同时 没想好啊

**Comment by 0xhelium** [comment]
此生误会入华夏
来世还做中国人(仅限港澳台地区)

**Reply by pengyidong** [reply]
你现在就想着下辈子做老爷，那你以后不得造反？

**Comment by clleo888** [comment]
如果中国人有权选择自己的政府, 共党早就不存在了!

**Reply by miaodi** [reply]
> 如果中国人有权选择自己的政府, 共党早就不存在了!

所以说中国人，汉人这个概念就是个伪概念。作为公民，没有任何公民权利只有义务，这和奴隶毫无区别。最可笑的是，拆腻死还乐在其中，这已经不是奴隶而是奴才。所以早该图图了

**Reply by Pinwee** [reply]
> > 如果中国人有权选择自己的政府, 共党早就不存在了!
> 
> 所以说中国人，汉人这个概念就是个伪概念。作为公民，没有任何公民权利只有义务，这和奴隶毫无区别。最可笑的是，拆腻死还乐在其中，这已经不是奴隶而是奴才。所以早该图图了

还是搞不懂为什么万恶资本主义公民挺滋润的，以民为本社会主义却很多水深火热。真害怕之后生活哪一部走错了，申诉维权的路子都没有，像那银行，烂尾楼。

**Comment by stevenwsg** [comment]
来世去做北欧人

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Reply by maxwillma** [reply]
你咋这么高兴？

**Reply by Cegazhos** [reply]
> 你咋这么高兴？

這是機器人。鄉巴佬少對著墻壁擼，見到顆樹都當媽
